src/test/java/com/trello/authorization/AuthorisationTest.java

```diff
@@ -17,27 +17,34 @@
 public class AuthorisationTest {
 
     public ChromeDriver driver;
-    WaitUtils waitUtils;
+    private LogInPage logInPage;
+    private BoardPage boardPage;
+    private User user;
 
-    String emailUser = "fesenko.b@icloud.com";
-    String passwordUser = "B0r1sTr3ll0";
+    String userEmail = "workboris3@gmail.com";
+    String userPassword = "B0r1sTr3ll0";
 
     @BeforeClass
     public void webDriver() {
         WebDriverManager.chromedriver()
                         .setup();
-        driver = new ChromeDriver();
+        user = new User(userEmail, userPassword);
     }
 
     @BeforeMethod
     public void openEnvironment() {
+        driver = new ChromeDriver();
         driver.get("https://trello.com");
+        logInPage = new LogInPage(driver);
+        boardPage = new BoardPage(driver);
+        logInPage.openLoginPageFactory(driver);
+
     }
 
     @AfterMethod
     public void signOffFromTrello() {
-        BoardPage boardPage = new BoardPage(driver);
         boardPage.logOutIfAuthorised();
+        driver.close();
```

Comment:
.quit()

Reply:
In these changes - I will delete the annotation @AfterClass (browser quit())